src/sass/layout-grid/layout-grid.scss

```diff
@@ -0,0 +1,253 @@
+@import "../variables/variables";
+
+// Recursive Mixin to build repetative class declarations for explicit dev overrides for various breakpoints
+@mixin layout-grid-templates($breakpoint, $repeatTo, $i: 1) {
+    @if $i <= $repeatTo {
+        .layout-grid[data-columns-#{$breakpoint}="#{$i}"] > ul {
+            grid-template-columns: repeat($i, 1fr);
+        }
+
+        @include layout-grid-templates($breakpoint, $repeatTo, ($i + 1));
+    }
+}
+
+.layout-grid {
+    --layout-grid-cell-height-min: 0;
+    --layout-grid-cell-gap: var(--spacing-100);
+    --layout-grid-columns-min: 1;
+    --layout-grid-columns-xs: 2;
+    --layout-grid-columns-sm: 3;
+    --layout-grid-columns-md: 4;
+    --layout-grid-columns-lg: 6;
+    --layout-grid-columns-xl: 8;
+    --layout-grid-columns-xl2: 10;
+    --layout-grid-columns-xl3: 12;
+    --layout-grid-columns-xl4: 14;
+
+    container: layout-grid-container / inline-size;
+}
+
+// smallest explicit dev display rule overrides
+// for default (any screen size) in the rules
+.layout-grid[data-columns-min="1"] > ul {
```

Comment:
Is there a reason we are containing this to ul only? 
Could we have a more generic version which uses divs or just classes maybe

I'm open mostly just checking. If we expect all items in a layout gird to be wrapped in a ul then it's fine

Reply:
IMO semantically, these should always be lists; that's why the previous components doing the layouts themselves also had only lists. The `ul` structure imposes the `a11y` for the group of items, which, if done with more generic elements, would need to manually add. I'm not sure having flexibility here is necessary or an improvement. The MIND pattern describes this as "group of list of items" as well. I'll run it by the MIND patterns team just to verify.

Reply:
If we can achieve everything visually with list markup, then I don't see any need to support div also. If there was a case where we needed to nullify the list semantics we could always add `role=none` to the root.